mobile/src/lib/device.js

```diff
@@ -0,0 +1,11 @@
+export const isCordova = () => (window.cordova !== undefined)
+export const isIos = () => (getDeviceName() === 'ios')
```

Comment:
I think the `const fun = () => {}` version is not equivalent as `fun() {}`: the first must be declared before it is used, not the second. Am I right?